docs/usage.rst

```diff
@@ -1265,6 +1265,119 @@ Besides the `invest` variable, new variables are introduced as well. These are:
       monthly periods, but you would need to be very careful in parameterizing your energy system and your model and also,
       this would mean monthly discounting (if applicable) as well as specifying your plants lifetimes in months.
 
+Modelling cellular energy systems
+---------------------------------
+
+The cellular approach is a concept proposed by the VDE-ETG. It is related to smart-grids and multi-microgrid systems but extends
+both. The idea is to group the components of an energy system into a hierarchically aggregating structure of cells. For example,
+the sources, sinks, storages and converters of a household could be a single cell. Then a group of physically neighboring 
+households could form another cell, made up of household-cells. This behaviour can be scaled up. The real game-changer in the 
+cellular approach is the way the cells are operated, which will not be covered here. Here, we focus on the way such cellular
+energy systems can be modeled.
+
+So far, the implementation in solph is just a neat way to group different parts of a larger energy system into cells. Further 
+functionality is expected to make its way into solph in the future, e.g. a distributed optimization procedure based on 
```

Comment:
You could mention that the structural updates are a prerequisite for most (or even any?) decomposition algorithm. I think, Bender's decomposition for instance would also be allowed for given the restructuring.